### ISSUE TITLE
combine exact same empty server

### ISSUE THREAD

**Issue by luk51** [ddnet/ddnet#5884]
When there are two servers (same location, eg both GER 2), hide one of them and indicate, that the row in the server browser list represents two empy servers.

* either like this: "2x DDNet GER 2 - Brutal"
* or like this (in Player count) "0/63x2"

Only combine empty servers.
Only exact same (same location, map, type, 0 players).
Add switch to activate / deactivate this feature in settings.

in this case it would change something:
![brutal-1](https://user-images.githubusercontent.com/4086396/192297387-18104b96-25ac-4111-b8a5-d31ac682e092.png)

**Comment by def-** [comment]
I don't think this is a good idea. They are still different servers, so why group them? Do you also want IP to be same and map too?

**Comment by luk51** [comment]
Yeah, I don't know how to code it. I looked a bit into <i>serverbrowser.cpp</i>, but I think it would be a relatively big change to the code to add this feature, for such a small visible change.
My thought behind this was, that a random user might just look for a map to play, where players are online. In my case, I often want to play dummy maps, and search an empty server. I even don't care which map it has, because I'll vote for a particular map that I want to try. So just seeing there are 5 empty GER 2 servers would be a nice feature for me (seeing it in one row in the serverbrowser).
I started this branch, to try a bit out, but it's a huge change 😅 : https://github.com/luk51/ddnet/tree/smartfilter
